src/luasql/exasol/luws.lua

```diff
@@ -32,6 +32,9 @@ local STATE_READMASK = "mask"
 local MAXMESSAGE = 65535 -- maximum WS message size
 local CHUNKSIZE = 2048
 local DEFAULTMSGTIMEOUT = 0 -- drop connection if no message in this time (0=no timeout)
+local WS_UPGRADE_REQUEST_TIMEOUT <const> = 5
+local WS_UPGRADE_RESPONSE_TIMEOUT <const> = 5
+local WS_SEND_FRAME_TIMEOUT <const> = 15
```

Comment:
You changed the timeout from 5 to 15. Is this intended?

General: This is third-party code. We should modify it only if really necessary to simplify integration of future changes.

Reply:
Good catch. That was a remainder of an experiment. I will revert this change.